ItemLevelDownloader/Providers/filesystemProvider.ts

```diff
@@ -39,12 +41,12 @@ export class FilesystemProvider implements models.IArtifactProvider {
             const folder = path.dirname(outputFilename);
             this.ensureDirectoryExistence(folder);
 
-            console.log("Downloading '%s' to '%s'", item.path, outputFilename);
+            Logger.logMessage('Downloading '+ item.path + ' to '+ outputFilename);
             const outputStream = fs.createWriteStream(outputFilename);
             stream.pipe(outputStream);
             stream.on("end",
                 () => {
-                    console.log(`Downloaded '${item.path}' to '${outputFilename}'`);
+                    Logger.logMessage(`Downloaded '${item.path}' to '${outputFilename}'`);
```

Comment:
good to be consistent with how you format the log string - use ${} everywhere or string concat... I think ${} looks cleaner...